Formula/mariadb.rb

```diff
@@ -151,6 +151,9 @@ def caveats; <<-EOS.undent
 
     To connect:
         mysql -uroot
+
+    Using brew services will place a plist file at #{ENV["HOME"]}/Library/LaunchAgents/#{plist_name}
```

Comment:
Okay sounds good and fair point. I can work on this, unless you would prefer to implement - let me know. Want me to close this PR for one with those changes or update this one?